sdk/cosmos/azure-cosmos-encryption/src/main/java/com/azure/cosmos/models/EncryptionBridgeInternal.java

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+import com.azure.cosmos.implementation.Document;
+import com.azure.cosmos.implementation.ItemDeserializer;
+import com.azure.cosmos.implementation.ResourceResponse;
+import com.azure.cosmos.implementation.Warning;
+
+import static com.azure.cosmos.implementation.Warning.INTERNAL_USE_ONLY_WARNING;
+
+/**
+ * This is an internal class in the encryption project
+ */
+public class EncryptionBridgeInternal {
```

Comment:
Please put the warning on this class level as well - I am seeing so many customers using our BridgeInternals.. we need to be more explicit..  like what we have in azure-cosmos

/**
 * DO NOT USE.
 * This is meant to be used only internally as a bridge access to classes in implementation
 **/
@Warning(value = INTERNAL_USE_ONLY_WARNING)

Reply:
good catch. thanks will do.